src/Microsoft.Diagnostics.EventFlow.Core/project.json

```diff
@@ -1,6 +1,7 @@
 {
   "version": "1.0.0-*",
   "dependencies": {
+    "Microsoft.Diagnostics.Correlation.Common": "1.0.0-preview1",
```

Comment:
I was planning to move EventFlow out of preview some time towards the end of November and start using it in Service Fabric templates. For that, we should not rely on any preview libraries. How close is the Correlation lib from crossing the boundary from preview to "production"?